### PR TITLE
moving android to the registry attributes

### PR DESCRIPTION
**Pull request by mlunadia**
moving android to the registry attributes

### CODE REVIEW

**Review by ChrsMark — APPROVED** [review]

File: docs/attributes-registry/android.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Android
+
+**Status**: [Experimental][DocumentStatus]
+
+**type:** `android`
+
+**Description**: The Android platform on which the Android application is running.
```
Comment (AlexanderWert) [inline docs/attributes-registry/android.md:7]: We usually do not include the above lines in the registry markdown files.

See for example https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/http.md for comparison.

File: docs/attributes-registry/android.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+<!-- semconv registry.android -->
```
Comment (trisch-me) [inline docs/attributes-registry/android.md:9]: Also registry doesn't need requirement level

**Review by trisch-me — APPROVED** [review]
lgtm

**Review by jsuereth — COMMENTED** [review]
Looks good pending, @AlexanderWert's suggestion on the registry markdown.

**Comment by mlunadia** [comment]
> There's an additional Android attribute in here: https://github.com/open-telemetry/semantic-conventions/blob/main/model/logs/mobile-events.yaml
> 
> Should we move that (and maybe also the iOS one) as part of this PR as well?
> 
> EDIT: Actually that would be yet another case being blocked on https://github.com/open-telemetry/build-tools/issues/242, because we would mix resource with semantic attributes.

@AlexanderWert should we go ahead and merge the PR as it is today to progress on the attributes that are not blocked?

**Comment by breedx-splk** [comment]
@mlunadia Looks like this needs a rebase. Also commenting to keep this open because I think we're close?

**Comment by trisch-me** [comment]
@mlunadia could you please update this PR with fields @AlexanderWert mentioned above and change the type to be generic `attribute_group`.

**Comment by ChrsMark** [comment]
@open-telemetry/specs-semconv-approvers do we need a changelog for this or we can label with "Skip Changelog"?

**Comment by trisch-me** [comment]
@ChrsMark yes, we don't need a changelog entry for refactoring, I have added a label

**Review by ChrsMark — APPROVED** [review]

**Review by breedx-splk — APPROVED** [review]

**Review by joaopgrassi — APPROVED** [review]

**Pull request merged**